.golangci.yml

```diff
@@ -33,8 +33,12 @@ issues:
       - stylecheck
     path: _test\.go
     text: "should not use underscores in Go names"
+  # Exclude gocritic paramTypeCombine errors since it could make code less readable
+  - linters:
+      - gocritic
+    text: "paramTypeCombine"
+  # TO-DO: handle the following issues on separate tickets.
```

Comment:
Do we still need this TO-DO or has it been addressed?

Reply:
Yes we still need a TO-DO for the rest of the ignored issues